src/js/components/Ballot/BallotElectionList.jsx

```diff
@@ -294,24 +288,24 @@ export default class BallotElectionList extends Component {
     const ballotElectionListUpcomingSorted = this.props.ballotElectionList.concat();
     // We want to sort ascending so the next upcoming election is first
     ballotElectionListUpcomingSorted.sort((a, b) => {
-      const election_day_text_A = a.election_day_text.toLowerCase();
-      const election_day_text_B = b.election_day_text.toLowerCase();
-      if (election_day_text_A < election_day_text_B) { // sort string ascending
+      const electionDaytextA = a.election_day_text.toLowerCase();
```

Comment:
Instead of "electionDaytextA" this should be "electionDayTextA". (and the same for electionDaytextB)